cmd/rcd/rcd.go

```diff
@@ -32,18 +41,209 @@ See the [rc documentation](/rc/) for more info on the rc flags.
 		if rcflags.Opt.Enabled {
 			log.Fatalf("Don't supply --rc flag when using rcd")
 		}
+
 		// Start the rc
 		rcflags.Opt.Enabled = true
 		if len(args) > 0 {
 			rcflags.Opt.Files = args[0]
 		}
+
+		if rcflags.Opt.WebUI {
+
+			// Get the latest release details
+			WebUIURL, tag, size := GetLatestReleaseURL()
+
+			// Load the file
+			exists := exists(tag + ".zip")
+			if !exists {
+				fmt.Println("Downloading webui binary. Please wait Size :" + strconv.Itoa(size))
+				if err := DownloadFile(tag+".zip", WebUIURL); err != nil {
+					panic(err)
+				} else {
+					println("Unzipping")
+					if err := os.RemoveAll("/webui"); err != nil {
+						fmt.Println("No previous downloads to remove")
+					}
+					if err := Unzip(tag+".zip", "webui"); err != nil {
+						panic("Error extracting file")
+					}
+				}
+			} else {
+				println("Files already exists. Skipping download")
+			}
+		}
+
 		s, err := rcserver.Start(&rcflags.Opt)
 		if err != nil {
 			log.Fatalf("Failed to start remote control: %v", err)
 		}
 		if s == nil {
 			log.Fatal("rc server not configured")
 		}
+
 		s.Wait()
 	},
 }
+
+/*
+GetLatestReleaseURL returns the latest release details of the rclone-webui-react
+*/
+func GetLatestReleaseURL() (string, string, int) {
```

Comment:
These helper functions don't need to be exported - start them with a small letter, eg `getLatestReleaseURL`